src/models/User.js

```diff
@@ -1,9 +1,12 @@
 const { Schema, model } = require('mongoose');
-const emailValidator = /^[a-zA-Z0-9._%+-]{3,}@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
 
+
+//TODO: add validation for fields in the schema
 const userSchema = new Schema({
```

Comment:
is this the final user model ?

Reply:
You are the boss, I think it is enough for now!

Reply:
We are a team, I'm not a boss :)
I think "city" and "likedEvents" array/array of objects, which are optional, are necessary. Also, as we talked, "isDeleted" boolean property is mandatory with default status.